### DISCUSSION TITLE
In the current version the placeholder field for a textarea changed back again to an input field instead of an textarea.

### DISCUSSION THREAD

**Discussion by martijn-dev** [directus/directus#12924] [Archive]
In the current version the placeholder field for a textarea changed back again to an input field instead of an textarea.

_Originally posted by @martijn-dev in https://github.com/directus/directus/issues/9003#issuecomment-1105010092_

**Comment by azrikahar** [comment]
I believe it's due to the change here: https://github.com/directus/directus/pull/12170/commits/4ecc06b62f8f3f02d3999c352dfea0e0a3ae6bf8#diff-ebbd14393f62040c06f9e8af93dd625bf7852ed27e687b7a96fb8a437e7daac9L20, to use the new translation strings input. @rijkvanzanten thoughts on this? Not sure what are the alternatives besides creating one more dedicated multiline variant of it 🤔

**Reply by rijkvanzanten** [reply]
Yup! I'd consider it an improvement to add a multiline input variant of the same translations input
